NorthstarDLL/mods/modmanager.cpp

```diff
@@ -410,6 +411,23 @@ void ModManager::LoadMods()
 			if (fs::exists(dir.path() / "mod.json"))
 				modDirs.push_back(dir.path());
 
+	// Special case for Thunderstore mods dir
+	std::filesystem::directory_iterator thunderstoreModsDir = fs::directory_iterator(GetThunderstoreModFolderPath());
+	for (fs::directory_entry dir : thunderstoreModsDir)
+	{
+		fs::path modsDir = dir.path() / "mods"; // Check for mods folder in the Thunderstore mod
```

Comment:
Additionally we could add a check here to enforce that the folder matches the `AUTHOR-MOD-VERSION` pattern (e.g. `GeckoEidechse-Titanfall_Slander-0.6.9`)

Reply:
This is a good idea. Make sure that if this is implemented, we don't just silently skip loading the package, but we log a warning. At some point I'd like to be able to keep track of all modloading warnings and display them in-game.

Reply:
Done in 7a6da3754753dab57ba5401c48f2d91484b93741. Mods that do not match the pattern get skipped and a warning is logged.

Example:
![image](https://github.com/R2Northstar/NorthstarLauncher/assets/40122905/c89650d3-e37e-448b-8395-33987f38976a)
gives
```
[01:09:29] [NORTHSTAR] [warning] The following directory did not match 'AUTHOR-MOD-VERSION' R2Northstar/packages\S2-Mods-GlitchOverhaul-1.2.11\mods
[01:09:29] [NORTHSTAR] [warning] The following directory did not match 'AUTHOR-MOD-VERSION' R2Northstar/packages\S2ModsGlitchOverhaul-1.2.11\mods
```